### DISCUSSION TITLE
Issue connecting to postal smtp no route to host

### DISCUSSION THREAD

**Discussion by Akazox** [postalserver/postal#2010] [Help with using Postal]
So i'm having this issue and i spent hours trying to fixe it as a complete newbie and i couldn't find the right solution.

I'm using contabo vps service so port 25 is open.
I tried using mautic, mailster, mailwizz to connect and always the same issue so i assume the problem is in my postal config maybe ?

Did anyone have this issue before ? 

thanks for your help
![image_2022-06-03_043928084](https://user-images.githubusercontent.com/105652645/171781496-6a47052e-fd75-4f31-896d-c7e8193f0442.png)

**Comment by willpower232** [comment]
If Postal is definitely running on your server then the answer is that something is blocking port 25, it could be a local or remote firewall or your server provider.
